go/teams/seitan.go

```diff
@@ -35,21 +35,24 @@ const seitanEncodedIKeyPlusOffset = 5
 const KBase30EncodeStd = "abcdefghjkmnpqrsuvwxyz23456789"
 const base30BitMask = byte(0x1f)
 
+type SeitanVersion uint
+
+const SeitanVersion1 = 1
```

Comment:
I think this is the way Go prefers it. For instance:

```go
type StatusCode int

const (
	StatusCode_SCOk                        StatusCode = 0
	StatusCode_SCInputError                StatusCode = 100
)
```

Reply:
done